src/mmw/mmw/settings/base.py

```diff
@@ -368,3 +368,16 @@ def get_env_setting(setting):
     'access_token_url': 'auth/concord_id/access_token',
     'user_json_url': 'auth/concord_id/user.json',
 }
+
+BASE_LAYERS = {
+    'Default': {
+        'url': 'https://{s}.tiles.mapbox.com/v3/ctaylor.lg2deoc9/{z}/{x}/{y}.png',
```

Comment:
Not to my knowledge on either, but they've expressed appreciation of the basemap we've been using.  We'll have to go through the rigmarole of explaining the options and the potential cost of using the MapBox one, unless they have a free tile set that's similar in style to this. I'll put it on the agenda for next week.